dotcom-rendering/src/web/components/LightboxPicture.tsx

```diff
@@ -0,0 +1,122 @@
+import { css } from '@emotion/react';
+import { breakpoints } from '@guardian/source-foundations';
+import React from 'react';
+import type { ImageWidthType } from './Picture';
+import { descendingByBreakpoint, generateImageURL } from './Picture';
+
+export type Props = {
+	master: string;
+	alt: string;
+};
+
+/**
+ * WIP: this should cover all use cases with a lot more precision
+ */
+const decideImageWidths = (): ImageWidthType[] => {
+	return [
+		{ breakpoint: breakpoints.wide, width: breakpoints.wide },
+		{ breakpoint: breakpoints.leftCol, width: breakpoints.leftCol },
+		{ breakpoint: breakpoints.desktop, width: breakpoints.desktop },
+		{ breakpoint: breakpoints.tablet, width: breakpoints.tablet },
+		{ breakpoint: breakpoints.phablet, width: breakpoints.phablet },
+		{
+			breakpoint: breakpoints.mobileLandscape,
+			width: breakpoints.mobileLandscape,
+		},
+		{
+			breakpoint: breakpoints.mobileMedium,
+			width: breakpoints.mobileMedium,
+		},
+		{ breakpoint: breakpoints.mobile, width: breakpoints.mobile },
+	];
+};
+
+/**
+ * Used on `picture` and `img` to prevent having a line-height,
+ * as these elements are which are `inline` by default.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#styling_with_css
+ */
+const block = css`
+	display: block;
+`;
+
+/** On fronts, all images are 5:3 */
+const aspectRatio = css`
+	padding-top: ${(3 / 5) * 100}%;
+	position: relative;
+	& > * {
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+	}
+`;
```

Comment:
TODO: This was copied from https://github.com/guardian/dotcom-rendering/pull/6065 and I should review if its what we want here.